include/wx/translation.h

```diff
@@ -346,6 +335,116 @@ inline const wxString& wxGetTranslation(const char *str1,
 
 #endif // wxNO_IMPLICIT_WXSTRING_ENCODING
 
+namespace wxTransImplStrict
+{
+
+// Wrapper functions that only accept string literals as arguments,
+// not variables, not char* pointers.
+template<size_t N>
+const wxString &wxUnderscoreWrapper(const char (&msg)[N])
```

Comment:
Extremely minor style nitpick, but could we please use this style:

```suggestion
const wxString& wxUnderscoreWrapper(const char (&msg)[N])
```

everywhere here?